graphql_compiler/compiler/emit_sql.py

```diff
@@ -913,10 +917,13 @@ def _wrap_into_cte(self) -> None:
         if self._recurse_needs_cte:
             self._current_alias = self.get_query(extra_outputs).cte(recursive=False)
             self._from_clause = self._current_alias
-
             self._filters = []  # The filters are already included in the cte
-            self._aliases = {
-                alias_key: ColumnRouter(
+
+            # After creating this CTE, accessing a field of any of the currently marked
+            # locations should redirect to the column of the cte that exposes that field.
```

Comment:
```suggestion
            # locations should redirect to the column of the CTE that exposes that field.
```

Reply:
2/3 uses of CTE in this block comment were capitalized, so I felt compelled to capitalize the last one :D